pkg/plugins/policies/core/matchers/testdata/egressmatchedpolicies/torules/01.golden.yaml

```diff
@@ -39,3 +39,4 @@ Rules:
     - Key: version
       Not: true
       Value: v2
+RulesNew: null
```

Comment:
should this appear in toRules?

Reply:
yes, the test checks for `policies.FromRules` https://github.com/kumahq/kuma/blob/3034ff67fb6a814242083fda083b66509598a21c/pkg/plugins/policies/core/matchers/egress_test.go#L109

it's egress, so toRules here are converted to rules per listener